infra/blueprint-test/pkg/kpt/kpt.go

```diff
@@ -0,0 +1,69 @@
+package kpt
+
+import (
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/mitchellh/go-testing-interface"
+)
+
+type CmdCfg struct {
+	kptBinary string         // kpt binary
+	dir       string         // dir to execute commands in
+	logger    *logger.Logger // custom logger
+	t         testing.TB     // TestingT or TestingB
+}
+
+type cmdOption func(*CmdCfg)
+
+func WithDir(dir string) cmdOption {
```

Comment:
Small note, but since it looks like the `dir`, `logger`, and `binary` functions are getting fairly repetitive across these utils, we could consolidate them into a shared, embedded `BinaryCfg` struct: https://golang.org/doc/effective_go#embedding

Reply:
I was planning to refactor these and TF/KRMBlueprintTests to use this pattern in a follow up PR

Reply:
Ack, sounds good.